include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -658,13 +658,14 @@ namespace internal
 
             requests.resize(requests.size() + 1);
 
-            MPI_Isend(buffer.data(),
-                      buffer.size(),
-                      Utilities::MPI::internal::mpi_type_id(buffer.data()),
-                      i.first,
-                      11,
-                      communicator,
-                      &requests.back());
+            MPI_Isend(
+              buffer.data(),
+              buffer.size(),
+              Utilities::MPI::internal::mpi_type_id(buffer.data()),
```

Comment:
Can you please spell out the type of `buffer` in line 646, which is `std::vector<types::global_dof_index>`? This helps to see the data type more explicitly.